cli/tests/test_all.rs

```diff
@@ -0,0 +1,624 @@
+use cid::Cid;
+use filecoindot_cli::{hex_encode_proof, NodeType};
+use filecoindot_proofs::deserialize_to_node;
+use filecoindot_proofs::traits::GetCid;
+use filecoindot_proofs::ProofVerify;
+use std::convert::TryFrom;
+
+#[test]
+fn verify_works() {
+    let p1 = vec![
```

Comment:
This is derived from print debugging the generated proof used in `forest_hamt_adaptor` test.